packages/@adobe/spectrum-css-temp/components/overlay/index.css

```diff
@@ -29,7 +29,7 @@ governing permissions and limitations under the License.
 %spectrum-overlay--open {
   visibility: visible;
 
-  opacity: 1;
+  opacity: 0.99;
```

Comment:
I'm not very happy about this fix, but only this worked for Edge to preserve same menu size when transitioning from 0 to 1 opacity. 

A stacking context is formed for opacity less then 1, and then its removed for 1. And I think it causes this rendering flicker that is visible when css transition is applied.

Reply:
Maybe try setting a `z-index` instead? I believe overlays are already `position: absolute`. See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context

Reply:
tried with `z-index` didnt work, also tried with:
`will-change`
`backface-visibility`
`transform-style: preserve-3d;`
`transformZ`
and other properties... nothing seems to work for Edge

since this bug is not applicable to Chromium, maybe to wait for Chromium to replace Edge?